src/core/library.h

```diff
@@ -203,11 +203,31 @@ typedef struct QUIC_LIBRARY {
     _Field_size_(PartitionCount)
     QUIC_PARTITION* Partitions;
 
+    //
+    // Lock protecting the stateless retry configuration.
+    //
+    CXPLAT_DISPATCH_RW_LOCK StatelessRetryLock;
```

Comment:
nit: Should we consider grouping all the members related to stateless retry in a struct to keep the library context cleaner?